build/lib/helm_require.sh

```diff
@@ -48,6 +48,8 @@ export IMAGE_TAG
 export HELM_REGISTRY
 envsubst <helm/sedfile.template >${SEDFILE}
 echo "s,version: v,version: ,g" >>${SEDFILE}
+echo "/^version:/s,-helm,," >>${SEDFILE}
```

Comment:
Yeh, not sure there is consensus on this, but I think we should get rid of `-helm`

Reply:
Will do